testsuite/tests/base_test/pkg.rst

```diff
@@ -0,0 +1,118 @@
+
+Pkg
+---
+
+.. ada:set_package:: Pkg
+
+Top-level package documentation.  I'm baby la croix drinking vinegar
+actually, photo booth pinterest raw denim coloring book occupy meggings
+church-key yr. Waistcoat tofu bruh mustache cornhole butcher normcore
+forage Brooklyn ramps. Sus lumbersexual PBR&B, cupping VHS swag tofu
```

Comment:
TIL: lumbersexual :axe:

Reply:
Yes, this comes from hipster ipsum https://hipsum.co/. I'm going to remove it because it's not very serious looking, even if it's a pretty benine word ;)